drakcore/drakcore/system.py

```diff
@@ -16,13 +17,25 @@ def __init__(self, path):
         super(SystemServiceConfig, self).__init__(path)
 
 
+
 class SystemService(KartonBase):
```

Comment:
Would be cool if the system service would be a part of karton itself, otherwise we have to copy paste it to our project :P

@psrok1

Reply:
+1 i had to steal it once again :D

Reply:
@chivay afaik this is fixed in the latest version